### PR TITLE
Increase weights for serial (en TN)

### PR DESCRIPTION
**Pull request by anand-nv**
# What does this PR do ?
Increase weights for serial (en TN)

**PR Type**:
- [x] Bugfix - Resolves https://github.com/NVIDIA/NeMo-text-processing/issues/126

### CODE REVIEW

File: tests/nemo_text_processing/en/data_text_normalization/test_cases_money.txt [hunk 1]
```diff
@@ -63,3 +63,4 @@ $1,925.21~one thousand nine hundred and twenty five dollars twenty one cents
 $1,234.123~one thousand two hundred and thirty four point one two three dollars
 US $76.3 trillion~US seventy six point three trillion dollars
 US$76.3 trillion~seventy six point three trillion us dollars
+The price for each canned salmon is $5, each bottle of peanut butter is $3~The price for each canned salmon is five dollars, each bottle of peanut butter is three dollars
```
Comment (ekmb) [inline tests/nemo_text_processing/en/data_text_normalization/test_cases_money.txt:66]: How did this one pass w/o Jenkins update?

Comment (anand-nv) [reply tests/nemo_text_processing/en/data_text_normalization/test_cases_money.txt:66]: Interesting 🤔.

**Review by tbartley94 — APPROVED** [review]
Nothing major so leaving approving review for quick merge.

File: tests/nemo_text_processing/en/data_text_normalization/test_cases_money.txt (hunk 1 above)
Comment (ekmb) [reply tests/nemo_text_processing/en/data_text_normalization/test_cases_money.txt:66]: seems like the added test case was working w/o weight update. Could you please replace with the input from [this issue](https://github.com/NVIDIA/NeMo-text-processing/issues/126)`Thank you for the quantities. Now, lets talk about the pricing. The price for each canned salmon is \$5, each bottle of peanut butter is \$3`

**Review by ekmb — CHANGES_REQUESTED** [review]

**Comment by anand-nv** [comment]
This test doesn't work correctly for me with the original weights. Not sure how it passed the tests.

________________________________

**Review by ekmb — APPROVED** [review]

**Pull request merged**